src/Database/Connection.php

```diff
@@ -448,9 +446,9 @@ public function update(string $table, array $values, array $conditions = [], arr
      * @param string $table the table to delete rows from
      * @param array $conditions conditions to be set for delete statement
      * @param array $types list of associative array containing the types to be used for casting
-     * @return \Cake\Database\StatementInterface
+     * @return \Cake\Database\Statement
```

Comment:
Why don't we want to return an interface here?

Reply:
There's no reason to support an Interface now that we don't use the nested decorator pattern. We expect a `Statement`.

Reply:
Re-introduced StatementInterface.